accord-core/src/main/java/accord/messages/WaitUntilApplied.java

```diff
@@ -170,6 +170,7 @@ private ReadNack apply(SafeCommandStore safeStore, SafeCommand safeCommand)
                 }
                 else
                 {
+                    // TODO (debug): this will cause us to send a Nack back, so why are we listening?
```

Comment:
will remove, but in debugger I wasn't sure why we do this... `NotCommitted` causes the process to return a `Nack` which causes the waiting to halt!  In my usage I keep retrying the wait over and over again when a `NotCommitted` or a `Timeout` is observed...